Add filtering of userlog based on settings (#10779)
owncloud/ocis: changelog/unreleased/enhancement-add-events-setting.md

> Enhancement: The user settings was extended by the notifications preferences

Reviewer: ```suggestion
Enhancement: The user settings are extended by notifications preferences
```

> The user settings was extended, that a user can store their notifications preferences

Reviewer: ```suggestion
Users can now store their notifications preferences in the user settings
```
